src/StatusQ/Components/StatusExpandableSettingsItem.qml

```diff
@@ -0,0 +1,150 @@
+import QtQuick 2.14
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+Rectangle {
+    id: root
+
+    property alias icon: icon
+    property alias primaryText: primaryText
+    property alias secondaryText: secondaryText
+    property alias button: button
+    property alias expandableComponent: expandableRegion.sourceComponent
+
+    property bool isExpandable: true
+    property bool hasButton: false
+    property  bool isExpanded: false
+    property bool hasDropDownArrow: false
+
+    anchors.left: parent.left
+    anchors.leftMargin: 20
+    anchors.right: parent.right
+    anchors.rightMargin: 20
```

Comment:
Please remove the `parent` dependency and rather give it an `implicitWidth`. Consumers of this component can then decide to anchors the component if they need to. The margins would go away as well.

Reply:
Agreed with Pascal. For example, now its impossible to use it with `Row`

Reply:
thanks for noticing this!